installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,215 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-mirroring-disconnected-v2"]
+= Mirroring images for a disconnected installation using the oc-mirror plugin V2
+include::_attributes/common-attributes.adoc[]
+:context: installing-mirroring-disconnected
+
+toc::[]
+
+Running your cluster in a restricted network without direct internet connectivity is possible by installing the cluster from a mirrored set of {product-title} container images in a private registry. This registry must be running at all times provided that the cluster is running. See the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/installing/disconnected-installation-mirroring#prerequisites_installing-mirroring-disconnected[Prerequisites] section for more information.
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror images to a mirror registry in your fully or partially disconnected environments. You must run oc-mirror from a system with internet connectivity in order to download the required images from the official Red Hat registries.
+
+[id="installation-oc-mirror-workflow_{context}"]
+== High level workflow
+The following steps outline the high-level workflow on how to use the oc-mirror plugin to mirror images to a mirror registry:
+
+. Create an image set configuration file.
+. Mirror the image set to the mirror registry by using one of the following methods:
+** Mirror an image set directly to the mirror registry.
+** Mirror an image set to disk, transfer the image set to the target environment, then upload the image set to the target mirror registry.
+. Configure your cluster to use the resources generated by the oc-mirror plugin.
+. Repeat these steps to update your mirror registry as necessary.
+
+// About the oc-mirror plugin
+include::modules/oc-mirror-about.adoc[leveloffset=+1]
+
+// About V2
+include::modules/oc-mirror-v2-about.adoc[leveloffset=+2]
+
+// oc-mirror compatibility and support
+include::modules/oc-mirror-support.adoc[leveloffset=+1]
+
+// About the mirror registry
```

Comment:
Suggestion: is it possible to put this in a annex section? 
In this position in the doc, it feels to me that we've already dug a bit into the oc-mirror specifics, to go back to discussing mirror registry. WDYT?

Reply:
Maybe a link to it from the pre-requisites section below this?

Reply:
Yep, as discussed it will be moved.